Objects/stringlib/unicode_format.h

```diff
@@ -1044,14 +1062,19 @@ formatteriter_next(formatteriterobject *it)
         if (conversion_str == NULL)
             goto done;
 
-        tuple = PyTuple_Pack(4, literal_str, field_name_str, format_spec_str,
-                             conversion_str);
+        PyStructSequence_InitType(&FormatterIterResultType, &formatter_iter_result_desc);
```

Comment:
Wait, do you initialize the type every time when create an instance?

Reply:
Sorry, I cannot find the appropriate `PyInit_{module}` function to initialize the type so at this point I was doing this so there some implementation to discuss. Where should I initialize the StructSeq?

Reply:
Sorry, I just found where `_string` is initialized and I am initializing the type in `PyInit__string` in f10d429. Please, correct me if this is not the correct place.